convex/updateRankingWithNG.ts

```
@@ -0,0 +1,401 @@
+    const startTime = Date.now();
+    
+    // Update cron status to running
+    await ctx.runMutation(api.updateRanking.updateCronStatus, {
+      name: "updateAllRankingsWithNG",
+      status: "running",
+    });
+
+    try {
+      // Get NG list
+      const ngList = await getNGList();
+      console.log(`NG list loaded: ${ngList.videoIds.length} video IDs, ${ngList.videoTitles.length} titles`);
```

ESLint

[error] Unexpected console statement.

(no-console)